### PR TITLE
SDIT-1130: 📝 Start to update documentation for prisoner search

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,81 @@
...
-[![Docker Repository on Quay](https://quay.io/repository/hmpps/hmpps-prisoner-search-indexer/status "Docker Repository on Quay")](https://quay.io/repository/hmpps/hmpps-prisoner-search-indexer)
-[![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://prisoner-search-indexer-dev.prison.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs)
+[![CircleCI](https://circleci.com/gh/ministryofjustice/hmpps-prisoner-search/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/hmpps-prisoner-search)
+[![Docker](https://quay.io/repository/hmpps/hmpps-prisoner-search/status)](https://quay.io/repository/hmpps/hmpps-prisoner-search/status)
+[![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://prisoner-search-dev.prison.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config)
+
+This README provides information on the prisoner search service.  For the indexer service that keeps the search
+up-to-date from Prison systems (NOMIS) please see the [indexer README](hmpps-prisoner-search-indexer/README.md).
```
Comment (petergphillips) [inline README.md:8]: this README is now about the search - with separate readme for indexer as most people will only be interested in the search API

File: hmpps-prisoner-search-indexer/README.md [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+The entire NOMIS prisoner base is retrieved and over several hours the other index is fully populated.
+
+Once the index has finished, if there are no errors then the (housekeeping cronjob)[#housekeeping-cronjob] will mark the
+index as complete and switch to the new index.
+
+If the index build fails - there are messages left on the index dead letter queue - then the new index will remain 
+inactive until the DLQ is empty. It may take user intervention to clear the DLQ if some messages are genuinely 
+unprocessable (rather than just failed due to e.g. network issues).
+
+#### Index switch
```
Comment (petergphillips) [inline hmpps-prisoner-search-indexer/README.md:41]: only reached this far today before I lost the plot.  Will carry on with the rest next week but thought might as well commit what I've done so far

Comment (petergphillips) [reply hmpps-prisoner-search-indexer/README.md:41]: below this is verbatim copied from the old search so needs amending

File: hmpps-prisoner-search-indexer/README.md [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+```
+If running the commands in a namespace other than dev then the `queue-url` can be obtained from the `prisoner-search-indexer-queue` secret.
+
+
+### Useful App Insights Queries
+
+#### General logs (filtering out the offender update)
+``` kusto
+traces
+| where cloud_RoleName == "prisoner-offender-search"
```
Comment (mikehalmamoj) [inline hmpps-prisoner-search-indexer/README.md:506]: A reminder to update these

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**